Vizzuality/heco-invest
# Layout for Authentication pages

This PR adds a simple layout for the authentication pages which use forms:  

- Sign in
- Sign up / invitation
- Forgot password

The verification link/account type selection/invite pages will be built separately (eg: [LET-172](https://vizzuality.atlassian.net/browse/LET-172)) and use a different layout. 

**Also in this PR**: 

- Sign up page now uses the standardised `Alert` component    
- New `Forms/FieldErrors` component to standardise input error messages  
- Some minor input colour standardisation to make sure they all match (including alerts/errors)  

## Testing instructions

It should respect the acceptance criteria defined in the task. 

However, being it's just a layout and there are currently no implemented pages that use it, one could mock one (or check it on storybook, although the display may not be the best there.)

## Tracking

[LET-213](https://vizzuality.atlassian.net/browse/LET-213)

## Screenshot
<img width="1680" alt="Screenshot 2022-03-28 at 12 00 54" src="https://user-images.githubusercontent.com/6273795/160386736-e265fcbb-e774-4b8f-8dcb-5adae9600db9.png">
